device-types/Cisco/C9120AXI-Q.yaml

```diff
@@ -0,0 +1,19 @@
+---
+manufacturer: Cisco
+model: Catalyst 9120AXI-Q
+slug: cisco-c9120axi-Q
+part_number: C9120AXI-Q
+u_height: 0
+is_full_depth: false
+comments: '[Cisco Catalyst 9120AX Series Access Points Data Sheet](https://www.cisco.com/c/en/us/products/collateral/wireless/catalyst-9120ax-series-access-points/datasheet-c78-742115.html)'
+interfaces:
+  - name: WiFi
+    type: ieee802.11ax
```

Comment:
Config interface names on the 9130:

wired0
wifi0
wifi1

"Physical" Interface Names:

GigabitEthernet0
Dot11Radio0
Dot11Radio1

Reply:
This is a C9120, Please, do advice where can we find these config interface names.
I only got something like:
![image](https://github.com/netbox-community/devicetype-library/assets/147031422/00c4477c-91bc-4418-8a7f-2869143a8eae)

and nothing close to:
```
wired0
wifi0
wifi1
```

Reply:
Fixed interface names based on DNAC and WLC Radio interface outputs.